### PR TITLE
Add PyShExC recipe

### PR DESCRIPTION
**Pull request by jfy133**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/PyShExC/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - antlr4-python3-runtime >=4.9.3,<4.10.dev0
+    - chardet
+    - jsonasobj >=1.2.1
+    - pyjsg >=0.11.10
+    - rdflib-shim
+    - shexjsg >=0.8.1
+
+test:
+  imports:
+    - PyShExC
```
Comment (jfy133) [inline recipes/PyShExC/meta.yaml:33]:
```suggestion
```

**Comment by jfy133** [comment]
@conda-forge/help-python I'm rather confused here, if I run `pip install PyShExC` and try rinning `import PyShExC` in python it loads correctly (so it does work outside of conda).

Looking through the azure logs, I don't see any _errors_ or anything (just a few warnings)... any suggestions why the tests are thus failing at the test step?

File: recipes/PyShExC/meta.yaml (hunk 1 above)
Comment (ocefpaf) [inline recipes/PyShExC/meta.yaml:33]:
```suggestion
    - pyshexc
```

Comment (jfy133) [inline recipes/PyShExC/meta.yaml:33]:
```suggestion
    - pyshexc
```

**Comment by jfy133** [comment]
That was a misunderstanding on my part then (I didn't realise the entire package got renamed), thanks @ocefpaf !

Ready for review now!

**Pull request merged**